### PR TITLE
some new block

### PR DESCRIPTION
**Pull request by JavidHatamov**

### CODE REVIEW

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -2,6 +2,61 @@ import './HomePage.scss';
 
 export const HomePage = () => {
 return(
-<></>
-)
-};
\ No newline at end of file
+<>
+<div className="QuestionBlog narrow-container">
```
Comment (JavidHatamov) [inline src/pages/HomePage/HomePage.tsx]: please move this code inside of HomePage's child component

File: src/pages/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
+
+footer{
+    background-color:  #F5F5F5;
+    height: 313px;
+    }
+    .Logo{
+        width: 20px;
+    }
+    .imageSocial{
```
Comment (Negodyaj) [inline src/pages/Footer/Footer.scss]: we use kebab-case for classes
I mean:
Logo -> logo
imageSocial -> image-social
SameDisplay -> same-display
and so on

please change it everywhere)

File: src/pages/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
+
+footer{
+    background-color:  #F5F5F5;
```
Comment (Negodyaj) [inline src/pages/Footer/Footer.scss]: please create "_variables.scss" file if you don't have it yet, and declare variables for colors there
after that you've got to use those variables in this file

File: src/pages/HomePage/HomePage.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.QuestionBlog{
```
Comment (Negodyaj) [inline src/pages/HomePage/HomePage.scss:1]: same comments)

File: src/pages/HomePage/components/Questions/Questions.tsx [hunk 5]
```diff
@@ -3,7 +3,60 @@ import './Questions.scss';
 export const Questions= () =>{
     return(
         <>
-        <h1>Questions</h1>
+        <div className="QuestionBlog narrow-container">
+        <div className="Blog">
```
Comment (Negodyaj) [inline src/pages/HomePage/components/Questions/Questions.tsx]: you must create Question component a the same level as Questions component (no need to add a new "components" folder)
also, you need to create an array for you questions inside of this component and use questions.map(...) to render all your Question components

also I think each question must have an answer. so you need to create some text for this)
and later we will do questions to be expandable

File: src/pages/HomePage/HomePage.tsx (hunk 1 above)
Comment (JavidHatamov) [reply src/pages/HomePage/HomePage.tsx]: ok ))

File: src/pages/Footer/Footer.scss (hunk 2 above)
Comment (JavidHatamov) [reply src/pages/Footer/Footer.scss]: ok ))

File: src/pages/HomePage/components/Questions/Questions.tsx [hunk 6]
```diff
@@ -1,35 +1,59 @@
...
-            <path d="M85 197.435C51 196.078 43 183.5 43 159H42V198L85 197.435Z" fill="url(#paint1_linear_22_941)"/>
-            <defs>
-            <linearGradient id="paint0_linear_22_941" x1="21.5" y1="159" x2="21.5" y2="198" gradientUnits="userSpaceOnUse">
-            <stop stop-color="#EAEAEA"/>
-            <stop offset="1" stop-color="#F5F5F5"/>
+  return (
+    <>
+      <div className="question-block narrow-container">
+        {questions.map((question, index) => (
+          <Question question={questions[index]} />
```
Comment (Negodyaj) [inline src/pages/HomePage/components/Questions/Questions.tsx:17]: you forgot about "key" attribute (https://reactjs.org/docs/lists-and-keys.html)

**Pull request merged**